packages/site/package.json

```diff
@@ -0,0 +1,86 @@
+{
+    "name": "@comet/brevo-site",
+    "version": "1.0.0",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/vivid-planet/comet",
+        "directory": "packages/site"
+    },
+    "license": "BSD-2-Clause",
+    "main": "lib/index.js",
+    "types": "lib/index.d.ts",
+    "files": [
+        "lib/*"
+    ],
+    "scripts": {
+        "build": "$npm_execpath run clean && tsc -p tsconfig.build.json",
+        "clean": "rimraf lib",
+        "dev": "tsc --watch --preserveWatchOutput -p tsconfig.build.json",
+        "generate-schema": "ts-node generate-schema.ts",
+        "generate-schema:watch": "chokidar \"src/\" -c \"$npm_execpath generate-schema\"",
+        "lint": "run-p lint:eslint lint:tsc",
+        "lint:eslint": "eslint --max-warnings 0 src/ package.json",
+        "lint:prettier": "npx prettier --check './**/*.{js,json,md,yml,yaml}'",
+        "lint:tsc": "tsc --noEmit"
+    },
+    "dependencies": {
+        "@comet/cms-site": "^7.7.0",
+        "@fontsource/roboto": "^5.1.0",
+        "@next/bundle-analyzer": "^14.2.18",
+        "@opentelemetry/api": "^1.9.0",
+        "@opentelemetry/auto-instrumentations-node": "^0.53.0",
+        "@opentelemetry/exporter-trace-otlp-http": "^0.55.0",
+        "@opentelemetry/sdk-node": "^0.55.0",
+        "cache-manager": "^5.7.6",
+        "graphql": "^15.9.0",
+        "ioredis": "^5.4.1",
+        "lru-cache": "^11.0.2",
+        "next": "^14.2.18",
+        "react": "^18.0.0",
+        "react-dom": "^18.0.0",
+        "redraft": "^0.10.2",
+        "rimraf": "^3.0.0",
+        "styled-components": "^6.1.13",
+        "swiper": "^11.1.15",
+        "ts-node": "^10.9.2"
+    },
+    "devDependencies": {
```

Comment:
The dev dependencies could probably be reduced as well.

Reply:
Also removed here: https://github.com/vivid-planet/comet-brevo-module/commit/bdea97e28347a60ebeddd8c3f812fdec96a34128